apps/native-ui-storybook/components/Heading/Heading.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Center, Heading } from '@utilitywarehouse/native-ui';
+import { Meta } from '@storybook/react';
+
+const HeadingBasic = ({ ...props }: any) => {
+  return (
+    <Heading {...props}>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+      labore et dolore magna aliqua.
+    </Heading>
+  );
+};
+
+HeadingBasic.argTypes = {
+  size: {
+    options: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'],
```

Comment:
is this the convention we want to use? 🤔 I thought we were using more presentational naming like Tshirt size or something referencing a scale rather than hierarchies

Reply:
I've kept it the same as what we have to avoid confusing devs when looking at the text styles in the designs. I'm all for moving to a size approach, but we need to coordinate.